chalice/app.py

```diff
@@ -7,12 +7,12 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from dss import BucketStage, Config, create_app
+from dss import DeploymentStage, Config, create_app
```

Comment:
Sounds good. After you merge I'll rebase against that and that should take care of any conflicts